src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,95 @@
+import cn from 'classnames';
+import React, { useState } from 'react';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const numberOfPages: number = Math.ceil(total / perPage);
+
+  const [activePage, setActivePage] = useState('1');
+
+  return (
+    <ul className="pagination">
+      <li
+        className={cn('page-item', {
+          disabled: currentPage === 1,
+          active: activePage === 'prev',
+        })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1 ? true : false}
+          onClick={() => {
+            if (currentPage !== 1) {
+              onPageChange(currentPage - 1);
+            }
+
+            setActivePage('prev');
```

Comment:
Setting `activePage` to 'prev' does not seem to be necessary, as the active page should be a number representing the current page, not a string indicating the direction of navigation.